pkg/cmd/server/api/types.go

```diff
@@ -1004,6 +1004,9 @@ type LDAPQuery struct {
 
 	// Filter is a valid LDAP search filter that retrieves all relevant entries from the LDAP server with the base DN
 	Filter string
+
+	// PageSize is the maxmimum preferred page size, measured in LDAP entries
+	PageSize int
```

Comment:
some servers might not support paging, might not allow the particular user to page, or might have min/max sizes incompatible with what we pick